src/app/cluster/details/external-cluster/template.html

```diff
@@ -105,9 +105,13 @@
             {{cluster?.id}}
           </div>
         </km-property>
-        <km-property *ngIf="provider !== Provider.Custom">
+        <km-property [ngSwitch]="provider !== Provider.Custom">
```

Comment:
i think this check needs to be fixed since now we need to add `provider`  in both Custom and Non Custom (External) providers.

It has effected change due to https://github.com/kubermatic/dashboard/pull/4846/files#diff-d09dc0d53644c292082db36ced11df2b0997bb07330d2892dd65bdcbd7bcfc6cR103

If see method `static getProvider` in `src/app/shared/entity/external-cluster.ts` you the `if` condition will never be executed now as `!cloud` will NEVER be empty in case of `Custom`
`{ bringyourown: {} }` from now onwards.


So coming back we don't need this CHECK ngSwitch anymore.

```suggestion
   <span class="km-provider-logo" [ngClass]="provider === Provider.Custom ?  'km-provider-logo-custom-externalcluster' :  'km-provider-logo-{{provider}}">
```

Reply:
@KhizerRehan It will still display in both cases because `*ngSwitchCase` is on icons and not the `Provider` text. However, I agree that code can be improved:

```suggestion
        <km-property>
          <div key>Provider</div>
          <div value>
            <span class="km-provider-logo" 
                        [ngClass]="provider === Provider.Custom ?  'km-provider-logo-custom-externalcluster' :  'km-provider-logo-' + provider"></span>
        </km-property>
```